marquez/client.py

```diff
@@ -0,0 +1,264 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+import urllib
+
+
+from marquez.constants import JobRunState
+from marquez.utils import request_utils
+from marquez.utils.request_utils import (InvalidRequestException,
```

Comment:
For simplicity, I recommend defining all convenience methods in a single `utils.py`. As more methods are added, breaking them up by functionality into separate utils might become a thing, but feels heavy to have `request_utils` so early on.

Reply:
Refactored it all into a single file :)